code/datums/jobs/job/shipside.dm

```diff
@@ -296,7 +296,7 @@ You are in charge of logistics and the overwatch system. You are also in line to
 	title = PILOT_OFFICER
 	paygrade = "WO"
 	comm_title = "PO"
-	total_positions = 2
+	total_positions = 1
```

Comment:
You forgot to reduce the PO slots in nuclear_war.dm so two PO's can still spawn in